js/functions.js

```diff
@@ -0,0 +1,67 @@
+// Функция для проверки длины строки
+const isLessOrEqual = (string, length) => string.length <= length;
```

Comment:
тут ошибка втом что обявлена переменная но она ни где не используется.
Исправляется очень просто: вызови функцию
```suggestion
const isLessOrEqual = (string, length) => string.length <= length;
isLessOrEqual('asdasdasd', 3)
```